qa/L0_long_running_stress/scenarios.py

```diff
@@ -263,7 +263,7 @@ def generate_sequence_data(self, trial, dtype, data_filename):
                 if dtype == np.float32:
                     res = float(res)
                 elif dtype == np.int32:
-                    # Do nothing
+                    res = int(res)
```

Comment:
Why do we need `int` cast here?

Reply:
Good find. Heads up that https://github.com/triton-inference-server/server/pull/3674 was just merged and addressed this with pass (an allowed no-op). Might want to rebase this PR off main.

Reply:
Ignore my comment above, looks like you did rebase. Pass might be clearer/more efficient, since it's a no-op, but really up to preference.